### ISSUE TITLE
ImportError for cloud module

### ISSUE THREAD

**Issue by e-caste** [lhelontra/tensorflow-on-arm#67]
My simple fix for tensorflow v1.14.0 (doesn't work in v1.13.0): in the site-packages folder find your tensorflow folder, inside it find contrib/__init__py. Open it and comment out the line where it says `if os.name != "nt" and platform.machine() != "s390x":
  from tensorflow.contrib import cloud`
Obviously I've done this only because I don't need Google cloud for my application.
I'm on a Raspberry Pi 3B+ with armv7l architecture using Python 3.5.3 if that matters.

**Comment by VincentChong123** [comment]
Hi @e-caste ,

>My simple fix for tensorflow v1.14.0 (doesn't work in v1.13.0):

Thanks for sharing your solution. Do you thin it will work on tf1.13.1? 
It seems not working for tf1.12 according to [#52](https://github.com/lhelontra/tensorflow-on-arm/issues/52)
I have similar issue at [#25168](https://github.com/tensorflow/tensorflow/issues/25168) 
Thanks. :)

**Comment by VincentChong123** [comment]
Hi @e-caste ,

>Do you thin it will work on tf1.13.1?
It works. Thanks.
```
/usb/venv/tf1.13/lib/python3.5/site-packages/tensorflow/contrib/__init__.py
#if os.name != "nt":
#  from tensorflow.contrib import cloud
```
My system infor
[tensorflow-1.13.1-cp35-none-linux_aarch64.whl](https://github.com/lhelontra/tensorflow-on-arm/releases) 
Linux firefly 4.4.52 HKT 2017 aarch64 aarch64 aarch64 GNU/Linux
rk3399

**Comment by tadam98** [comment]
> My simple fix for tensorflow v1.14.0 (doesn't work in v1.13.0): in the site-packages folder find your tensorflow folder, inside it find contrib/__init__py. Open it and comment out the line where it says `if os.name != "nt" and platform.machine() != "s390x": from tensorflow.contrib import cloud`
> Obviously I've done this only because I don't need Google cloud for my application.
> I'm on a Raspberry Pi 3B+ with armv7l architecture using Python 3.5.3 if that matters.

This Works ! Why not update the buster whl of V1.14.0 to reflect this. ?
Best,
Mickey
